recipes-bsp/optee-os/optee-os-iot2050_3.18.0.bb

```diff
@@ -34,7 +35,3 @@ python do_patch_prepend() {
     if re.search("rpmb-setup", overrides) and re.search("secureboot", overrides):
         bb.fatal("Not possible to use Secure Boot and RPMB setup for OPTEE")
 }
```

Comment:
This could also be refined to inside an anonymous function.

Reply:
You are right - separate commit, will add it.